ui/packages/catalog-ui-search/src/main/webapp/react-component/inputs/enum-input/enum-input.js

```diff
@@ -60,16 +79,41 @@ const EnumInput = ({
           )}
         {filteredSuggestions.map(suggestion => {
           return (
-            <EnumMenuItem key={suggestion.value} value={suggestion.value}>
+            <EnumMenuItem
+              title={
+                isAttributeDisabled(settingsModel, suggestion)
+                  ? `Attribute is unsupported by the ${SOURCES} selected`
+                  : ''
+              }
+              key={suggestion.value}
+              value={suggestion.value}
+              disabled={isAttributeDisabled(settingsModel, suggestion)}
+            >
```

Comment:
This logic still feels like it should be raised to a higher level. I would expect each `suggestion` object to provide all these values, something like this (with some sensible defaults if they aren't set)
```
            <EnumMenuItem
              title={suggestion.title}
              key={suggestion.value}
              value={suggestion.value}
              disabled={suggestion.disabled}
            >

```